### DISCUSSION TITLE
Radio Button in style of Button

### DISCUSSION THREAD

**Discussion by XuanShine** [zauberzeug/nicegui#3306] [Q&A] [answered]
### Question

Hi,
How can I make a group of buttons that are bind together so the color change when another button of the same group is clicked ?
Like a Radio Button.
Is it possible to bind_color ?
Thanks

**Comment by python-and-novella** [comment, marked as answer]
toggle?
```python3
from nicegui import ui

toggle1 = ui.toggle([1, 2, 3], value=1)
toggle2 = ui.toggle({1: 'A', 2: 'B', 3: 'C'}).bind_value(toggle1, 'value')

ui.run()
```
![image](https://github.com/zauberzeug/nicegui/assets/16256910/cd0d4fa4-dc3f-4751-8dcc-4e97f1704d92)

**Reply by XuanShine** [reply]
Yes exactly... how could I missed that -_-
Thank you !
Ps: I was thinking about create a new Class of group-button to recreate the wheel xD
